consensus/abci.go

```diff
@@ -225,81 +203,70 @@ func (abci *TendermintABCI) validateTx(tx *ethTypes.Transaction) tmtAbciTypes.Re
 	// Make sure the transaction is signed properly
 	from, err := ethTypes.Sender(signer, tx)
 	if err != nil {
-		return tmtAbciTypes.ResponseCheckTx{Code: uint32(abciTypes.ErrBaseInvalidSignature.Code),
-			Log: core.ErrInvalidSender.Error()}
+		abci.logger.Error(core.ErrInvalidSender.Error())
```

Comment:
Did you check that we are not logging same error twice? once on this level and another again on Tendermint as a consequence of the response?

Reply:
Only one. error log was logged as far I saw, from previous 0, or worst, "Transaction successful" in case of an error hehe